ckan/cli/cli.py

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
+
+import os
+
+import click
+
+from ckan.cli import db, load_config, search_index, server, click_config_option
+from ckan.config.middleware import make_app
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class CkanCommand(object):
+
+    def __init__(self, conf=None):
+        self.config = load_config(conf)
+        self.app = make_app(self.config.global_conf, **self.config.local_conf)
+
+
+@click.group()
+@click.help_option(u'-h', u'--help')
+@click_config_option
+@click.pass_context
+def ckan(ctx, config, *args, **kwargs):
+    ctx.obj = CkanCommand(config)
+
+
+ckan.add_command(server.run)
```

Comment:
We probably want to automate this somehow and add support for extensions here, but we can do both these things at a later stage